nix/sources.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This file looks pretty copy-pasted, can't nix fetch it from somewhere itself?

Reply:
this pins the dependencies, they are entirely specific to this project.